packages/compass-web/src/entrypoint.tsx

```diff
@@ -59,11 +62,23 @@ import { useCompassWebLoggerAndTelemetry } from './logger-and-telemetry';
 import { type TelemetryServiceOptions } from '@mongodb-js/compass-telemetry';
 import { WorkspaceTab as WelcomeWorkspaceTab } from '@mongodb-js/compass-welcome';
 
-const WithAtlasProviders: React.FC = ({ children }) => {
+const WithAtlasProviders: React.FC<{
+  projectId: string;
+}> = ({ children, projectId }) => {
   return (
     <AtlasCloudAuthServiceProvider>
       <AtlasServiceProvider>
-        <AtlasAiServiceProvider>{children}</AtlasAiServiceProvider>
+        <AtlasAiServiceProvider
+          apiURLPreset="cloud"
+          urlConfig={{
+            'user-access': (userId: string) =>
+              aiURLConfig.cloud['user-access'](userId, projectId),
+            query: aiURLConfig.cloud.query(projectId),
+            aggregation: aiURLConfig.cloud.aggregation(projectId),
+          }}
```

Comment:
I think we touched in tech design on this, you don't really need to expose the actual config here if you're passing the url preset, the `projectId` also doesn't need to be passed to the component, it's an information available for every connection through connection info provider, so the only thing you need to pass here specifically is the preset type, baking as much of this domain specific logic into the ai provider

Reply:
Sounds good, thanks for calling it out (again sorry), I moved the configuration of URLs into where they're being used more. I was thinking its better for the logic around the endpoint to happen in the actual service and not the provider. How does that look? This way there's a smaller interface also. I ended up passing in the connection info and not moving around the providers. I think both work so we could always switch it out if we find it more useful to pass the connection info ref in instead.

Reply:
> I was thinking its better for the logic around the endpoint to happen in the actual service and not the provider. How does that look?

Yeah, makes absolute sense, looks good!